alpine/distributionscanner.go

```diff
@@ -32,6 +32,7 @@ const (
 const (
 	issuePath = `etc/issue`
 
+	edgeVersion    = `edge`
```

Comment:
can we consolidate both of these `const`s?

Reply:
not sure I follow

Reply:
@crozzy do you have any updates?

Reply:
I meant have one `const` block instead of 2

Reply:
oh didn't see that, will do

Reply:
should be good now